lib/alt/index.js

```diff
@@ -0,0 +1,366 @@
+"use strict";
```

Comment:
I'm thinking we don't check these files in but rather just have them ran on pre-publish